ballista/rust/executor/src/executor_server.rs

```diff
@@ -261,6 +262,8 @@ impl<T: 'static + AsLogicalPlan, U: 'static + AsExecutionPlan> TaskRunnerPool<T,
         let executor_server = self.executor_server.clone();
         tokio::spawn(async move {
             info!("Starting the task runner pool");
+            //TODO make it configurable
+            let dedicated_executor = DedicatedExecutor::new("task_runner", 4);
```

Comment:
I think this should default to the number of available CPU cores.

Reply:
set default to 4 due to: in ballista conf  `concurrent_tasks` set 4 . We will make it configurable